### DISCUSSION TITLE
AKS Single Sign On (SSO) in v0.0.9!

### DISCUSSION THREAD

**Discussion by weinong** [Azure/kubelogin#60] [Show and tell]
Thanks for @phealy who contributed #54, now available in v0.0.9 release, you can access the cluster without additional login prompt!

![render1615750657558](https://user-images.githubusercontent.com/4204090/111082072-94827980-84c3-11eb-9cc7-4a74b27c4fe5.gif)

**Comment by phealy** [comment]
Thanks for adding this in! It's a feature I thought I'd love having (and I definitely did once I added it 👍🏻) and I wanted others to be able to have it as well.
